ledger/participant-state-index/src/main/scala/com/daml/ledger/participant/state/index/v2/package.scala

```diff
@@ -0,0 +1,159 @@
+// Copyright (c) 2019 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package com.daml.ledger.participant.state.index
+
+import java.time.Instant
+
+import akka.NotUsed
+import akka.stream.scaladsl.Source
+import com.digitalasset.daml.lf.data.Ref
+import com.digitalasset.daml.lf.data.Time.Timestamp
+import com.digitalasset.daml.lf.transaction.{BlindingInfo, GenTransaction}
+import com.digitalasset.daml.lf.value.Value
+import com.digitalasset.daml.lf.value.Value.{AbsoluteContractId, NodeId}
+import com.digitalasset.ledger.api.domain._
+
+package object v2 {
+
+  final case class AcsUpdate(
+      optSubmitterInfo: Option[SubmitterInfo],
+      offset: LedgerOffset,
+      transactionMeta: TransactionMeta,
+      transactionId: TransactionId,
+      events: List[AcsUpdateEvent]
+  )
+
+  sealed trait AcsUpdateEvent extends Product with Serializable
+
+  object AcsUpdateEvent {
+
+    final case class Create(
+        transactionId: TransactionId,
+        nodeId: NodeId,
```

Comment:
do we actually need this? it looks like we don't need the eventId at all here: https://github.com/digital-asset/daml/pull/1314/files#diff-187da506c796f10dc0d43c95b946dd3fR37

Reply:
Discussed with @dajmaki: Needs to be an opaque `String` that actually comes from the `ReadService`.